cmd/errorformat/main.go

```diff
@@ -161,3 +199,42 @@ func (t *TrackingWriter) WriteNL() (int, error) {
 func (t *TrackingWriter) NeedNL() bool {
 	return t.last != '\n'
 }
+
+type CheckStyleWriter struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type CheckStyleWriter should have comment or be unexported